docs/experimental/extendable-param-files.md

```diff
@@ -0,0 +1,63 @@
+# Using the Extendable Bicep Parameters Feature (Experimental!)
+
+## What is it?
+
+Extendable Bicep Parameter Files is a feature that allows you to extend `.bicepparam` files from another `.bicepparam` file in order to reuse parameters across multiple deployments.
+
+When using extendable bicep parameter files, you will have a `base.bicepparam` file that can be used by multiple `.bicep` and `.bicepparam` files.
```

Comment:
If I read this, it sounds like the file _has to_ be called `base.bicepparam` specifically, but this is just an example right? I am also not sure what it means to be used by multiple `.bicep` and `.bicepparam` files. Isn't this file only going to be consumed by another `.bicepparam` file?

Reply:
I suggested a small change, do you think it addresses your comment?

Reply:
Yes, base.bicepparam is just an example. For the second part, yes it is only going to be consumed by another .bicepparam file, my wording was too confusing so I just took the whole last sentence out so they can just refer to the "Using" part of the article